<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
    <widget class="QWidget" name="train"/>
    <widget class="QWidget" name="user">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>311</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="info_stk"/>
      <widget class="QWidget" name="pass_stk"/>
      <widget class="QWidget" name="history_stk"/>
      <widget class="QWidget" name="money_stk">
       <widget class="QLabel" name="money_text">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>账户余额：</string>
        </property>
       </widget>
       <widget class="QLabel" name="money_dis_lab">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>110</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="money_text2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>110</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>元</string>
        </property>
       </widget>
       <widget class="QToolButton" name="money_query_btn">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>180</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
       <widget class="QToolButton" name="money_update_btn">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>充值</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="info_pb">
         <property name="text">
          <string>基本信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pass_pb">
         <property name="text">
          <string>乘客管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="his_pb">
         <property name="text">
          <string>我的车票</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="bal_pb">
         <property name="text">
          <string>账户余额</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="train_btm">
       <property name="text">
        <string>车次</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="user_btm">
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
